### PR TITLE
[chore] Add build ci

### LINKED ISSUE #13: ✨ [新機能] - ビルドのCIを入れる
### 新機能の概要

_No response_

### 詳細な説明

_No response_

### 関連するイシュー

_No response_

### 追加情報

- [ ] UI/UXの変更が必要
- [ ] ドキュメントの更新が必要
- [ ] テストが必要

### PR DESCRIPTION
**Pull request by SEKI-YUTA**
Closes #13

## 概要
Next.jsのビルドをするCIを作成しました。

## 変更点
Next.jsのビルドをするCIを作成しました。

### CODE REVIEW

File: .github/workflows/build_next.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          registry-url: "https://registry.npmjs.org"
+          cache: "pnpm"
+
+      - name: Install node_modules
+        run: |
+          node --version
+          pnpm install
+
+      - name: Build Next.js app
+        run: pnpm run build
```
Comment (SEKI-YUTA) [inline .github/workflows/build_next.yml:34]: `pnpm add -g next`を実行してから`next build`を実行したらなぜかビルドが失敗するので`pnpm run build`を実行してる

**Comment by SEKI-YUTA** [comment]
<summary>next buildを実行した際のエラーの内容<details>
next build
  ▲ Next.js 14.2.6

   Creating an optimized production build ...
 ✓ Compiled successfully
 ✓ Linting and checking validity of types    
 ✓ Collecting page data    
   Generating static pages (0/5)  [    ]TypeError: Cannot read properties of null (reading 'useContext')
    at exports.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/react@18.3.1/node_modules/react/cjs/react.production.min.js:24:495)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/pages/_error.js:1:6026)
    at Wc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at bd (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:77:404)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:217)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:71:145)

Error occurred prerendering page "/500". Read more: https://nextjs.org/docs/messages/prerender-error

TypeError: Cannot read properties of null (reading 'useContext')
    at exports.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/react@18.3.1/node_modules/react/cjs/react.production.min.js:24:495)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/pages/_error.js:1:6026)
    at Wc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at bd (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:77:404)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:217)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:71:145)
TypeError: Cannot read properties of null (reading 'useContext')
    at exports.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/react@18.3.1/node_modules/react/cjs/react.production.min.js:24:495)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/pages/_error.js:1:6026)
    at Wc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at bd (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:77:404)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:217)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:71:145)

Error occurred prerendering page "/404". Read more: https://nextjs.org/docs/messages/prerender-error

TypeError: Cannot read properties of null (reading 'useContext')
    at exports.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/react@18.3.1/node_modules/react/cjs/react.production.min.js:24:495)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/pages/_error.js:1:6026)
    at Wc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at bd (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:77:404)
    at Z (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:217)
    at $c (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
    at Zc (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:71:145)
   Generating static pages (3/5)  [=   ]TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546) {
  digest: '4290658484'
}
TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)

Error occurred prerendering page "/_not-found". Read more: https://nextjs.org/docs/messages/prerender-error

TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546) {
  digest: '4290658484'
}
TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error

TypeError: Cannot read properties of null (reading 'useContext')
    at t.useContext (/Users/sekiyuuta/Desktop/apps/next/shikashiru/node_modules/.pnpm/next@14.1.1_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:108898)
    at d (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:34424)
    at h (/Users/sekiyuuta/Desktop/apps/next/shikashiru/.next/server/chunks/192.js:8:26313)
    at nj (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:46251)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47571)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
    at nI (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47010)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:47717)
    at nM (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:61546)
    at nN (/Users/sekiyuuta/Library/pnpm/global/5/.pnpm/next@14.2.6_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:64546)
 ✓ Generating static pages (5/5)

> Export encountered errors on following paths:
        /_error: /404
        /_error: /500
        /_not-found/page: /_not-found
        /page: /

</details>
</summary>

**Comment by SEKI-YUTA** [comment]
@illionillion

File: .github/workflows/build_next.yml (hunk 1 above)
Comment (illionillion) [reply .github/workflows/build_next.yml:34]: なぜ`pnpm add -g next` でグローバルにNext.jsをインストールする必要があるのでしょうか？
ローカルにあるNext.jsを実行してビルドするのに`pnpm run build`を実行するので正しいと思います。

File: .github/workflows/build_next.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Build Next.js app
+
+on:
+  pull_request:
```
Comment (illionillion) [inline .github/workflows/build_next.yml:6]: ES Lintの方と同じで`main`と`dev`ブランチ向けのプルリクエストとプシュのみ実行されるようにしてください。

Comment (illionillion) [inline .github/workflows/build_next.yml]: この提案を取り込めばすぐに修正されます。

```suggestion
on:
  push:
    branches: ["main", "dev"]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: ["main", "dev"]
```

Comment (illionillion) [reply .github/workflows/build_next.yml]: `Commit suggestion`

File: .github/workflows/build_next.yml (hunk 1 above)
Comment (SEKI-YUTA) [reply .github/workflows/build_next.yml:34]: ローカル環境で試したところグローバルにnextをインストールしていない状態だとnextコマンドがみつからない旨のエラーが出て実行できなかったのでグローバルにインストールする必要があるのかなと思いました。
（上記は`pnpm install`を実行したあとで試しました。）

File: .github/workflows/build_next.yml (hunk 2 above)
Comment (SEKI-YUTA) [reply .github/workflows/build_next.yml]: 後で修正します！

Comment (SEKI-YUTA) [reply .github/workflows/build_next.yml]: @illionillion 
ご指摘の点を修正しました。

File: .github/workflows/build_next.yml (hunk 1 above)
Comment (illionillion) [reply .github/workflows/build_next.yml:34]: ローカルにインストールしているライブラリを実行するには`npx next build`か`pnpm next build`で実行できます。

**Review by illionillion — APPROVED** [review]

File: .github/workflows/build_next.yml (hunk 1 above)
Comment (SEKI-YUTA) [reply .github/workflows/build_next.yml:34]: package.jsonのscriptsには`next build`と書いていたので、それを実行すればいいのかなと思ってました。

**Pull request merged**